node_modules/gh-orgunit/tests/test-orgunit.js

```diff
@@ -908,6 +908,49 @@ describe('Organisational units', function() {
                 });
             });
         });
+
+        /**
+         * Test that verifies that the organisational unit a series is borrowed from is included in the response
+         */
+        it('verify the organisational unit a series is borrowed from is included in the response', function(callback) {
+            // Generate a structure where a series is borrowed under multiple organisational units
+            OrgUnitTestUtil.createSimpleTree(global.tests.admins.cam2013.client, function(courseUnit, subjectUnit, partUnit, moduleUnit1, series) {
+                OrgUnitTestUtil.assertCreateOrgUnit(global.tests.admins.cam2013.client, '#2', 'module', null, function(moduleUnit2) {
+                    OrgUnitTestUtil.assertCreateOrgUnit(global.tests.admins.cam2013.client, '#2', 'module', null, function(moduleUnit3) {
+                        var seriesToAdd = series[0];
+                        OrgUnitTestUtil.assertAddOrgUnitSeries(global.tests.admins.cam2013.client, moduleUnit2.id, [seriesToAdd.id], function() {
+                            OrgUnitTestUtil.assertAddOrgUnitSeries(global.tests.admins.cam2013.client, moduleUnit3.id, [seriesToAdd.id], function() {
+
+                                OrgUnitTestUtil.assertGetOrgUnits(global.tests.admins.cam2013.client, null, true, null, null, 'module', function(orgUnits) {
+
+                                    // There's no borrowing taking place in module 1 as it was the first organisational
+                                    // unit to add the series. The `borrowedFrom` key should not be present
+                                    var moduleUnit1Response = _.find(orgUnits.results, {'id': moduleUnit1.id});
+                                    var series = _.find(moduleUnit1Response.Series, {'id': seriesToAdd.id});
```

Comment:
I don't think so, we only return it:
 - from `/api/series/:id` if the `includeOrgUnits` flag is set to `true`
 - from /api/orgUnits if the `includeSeries` flag is set to `true`